client/state/ui/language/actions.js

```diff
@@ -20,7 +20,7 @@ import { LOCALE_SET } from 'state/action-types';
  * @returns {Object} Action
  */
 export const setLocale = ( localeSlug, localeVariant = null ) => {
-	switchLocale( localeSlug, localeVariant );
+	switchLocale( localeVariant || localeSlug );
```

Comment:
why not also adjust `setLocale` here which appears to do the same thing?

Reply:
Mostly because of the Redux `LOCALE_SET` action that we're returning below (which still uses both args). 

I might tackle that in a subsequent pass tho, see my note in the PR desc:

> My thinking is that we should only store the variant (naming it `localeSlug` in the reducer should be fine tho), and modify our selectors so there's one that returns the variation, and one that returns the variation-free locale slug -- both of them however evaluating the same information in the Redux tree.

Reply:
(I'm essentially climbing up the call stack ladder, moving the fallback setting further and further up until I can hopefully rip it out altogether.)